src/app/game/game.component.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { Observable, from, of, defer } from 'rxjs';
+
+import { GameComponent } from './game.component';
+import { Answer } from '../models/answer';
+import { AnswerService } from '../services/answer.service';
+
+const mockAnswer = { id: 1, word: 'ADEPT' } as Answer;
+
+const answerServiceStub = {
+  getAnswer() {
+    return of(mockAnswer);
+  }
+};
+
+describe('GameComponent', () => {
+  let component: GameComponent;
+  let fixture: ComponentFixture<GameComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [GameComponent],
+      providers: [{ provide: AnswerService, useValue: answerServiceStub }]
```

Comment:
A common pattern we tend to use here that helps maintain type safety (sort of) is this:

```typescript
providers: [{
  provide: AnswerService,
  useValue: <Partial<AnswerService>>{
      getAnswer: jest.fn().mockReturnValue(of(mockAnswer))
  }
}]
```

There's nothing particularly wrong with the way you did it, but this way you are required to specify the correct types for the mocked methods. Just an FYI 🙂

Reply:
This is great and much more Jest-y, thank you!